image-specs/el7.3-base

```diff
@@ -0,0 +1,16 @@
+#base=libguestfs:centos-7.3
+#name=el7.3-base
+#osinfo=centos7.3
+#distro=el7
+#version=v1
+#arch=x86_64
+#expand=/dev/sda2
+#lvexpand=/dev/cl/root
+#root-partition=/dev/cl/root
+root-password password:123456
+update
+install qemu-guest-agent
```

Comment:
We still use it when taking snapshots (vm.py line 441).
It's needed by 'VIR_DOMAIN_SNAPSHOT_CREATE_QUIESCE' which marks that the guest's
fs should be frozen when taking a snapshot.